pkg/reporting/cookbooks_test.go

```diff
@@ -234,7 +234,7 @@ func TestCookbooks(t *testing.T) {
 	c, err := subject.NewCookbooks(
 		newMockCookbook(cookbookList, nil, nil),
 		makeMockSearch(mockedNodesSearchRows(), nil),
-		false,
+		true,
```

Comment:
This is because we weren't testing the extended data?

Reply:
Yes.